client/src/pages/Login/Login.tsx

```diff
@@ -36,27 +38,44 @@ export default function Login(): JSX.Element {
   };
 
   return (
-    <Grid container component="main" className={classes.root}>
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+    <Grid container component="main" overflow={'hidden'} className={classes.root}>
+      {/* add img and style to the login nav abit (not on this ticket but need it visually ) */}
+      <Grid item xs={12} marginBottom={8} bgcolor={'white'} boxShadow={'light'} display={'inline-flex'}>
```

Comment:
`sx` property would be better suited for this.

Reply:
Applicable throughout the files if you can find different areas like on `Box`, `Grid item`, etc.